styles.css

```diff
@@ -0,0 +1,872 @@
+}
+
+.main__outer-container {
+    display: flex;
+    /*let op hier row in column verandert voor de triangles!!!*/
+    flex-direction: column;
+    justify-content: center;
+    /*align-items: stretch;*/
+    width: 100%;
+    /*let op hier position relative toegevoegd voor triangles!!!*/
+    position: relative;
+}
+
+.main__cta-button,
+.main__cta-button--footer {
+    background-color: var(--main-button-color);
+    padding: 40px 90px;
+    margin: 20px;
+    border: none;
+    border-radius: 60px;
+    color: var(--main-font-color-light);
+    text-transform: uppercase;
+    cursor: pointer;
+    /*font-size: 12px;*/
+    font-size: 29px;
+    box-shadow: 0 8px 10px var(--main-shadow-focus-color);
+    z-index: 1;
+}
+
+.main__cta-button--footer {
+    padding: 45px 140px;
+}
+
+/* ------------------------------- 4. Areas --------------------------------------- */
+
+/* --------------- 4.1 Header ------------------- */
+
+.header__outer-container,
+.projects__header--outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: stretch;
+    background-image: var(--main-gradient-color);
+    color: var(--main-font-color-light);
+}
+
+.header__inner-container,
+.projects__header--inner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 67vh;
+}
+
+
+/* --------------- 4.1b Header-Projects-page ------------------- */
+
+.projects__header--inner-container {
+    justify-content: center;
+    height: 98vh;
+}
+
+/* --------------- 4.2 Header-Navigation ------------------- */
+.navigation {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navigation__list {
+    list-style-type: none;
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    gap: 2em;
+    margin: 4em 2em;
+    width: 100%;
+}
+
+.navigation___link:visited{
+    color: var(--main-font-color-light);
+}
+
+/* --------------- 4.3 Header-Hero-section ------------------- */
+
+.header__inner-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.header__hero-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    position: relative;
+    top: -42px;
+    z-index: 1;
+}
+
+.header__hero-section > h1 {
+    text-align: center;
+}
+
+.header__hero-section-paragraph {
+    font-size: 54px;
+    font-weight: 300;
+    text-align: center;
+    width: 100%;
+    margin: 1em;
+}
+
+.header__squiggle-line {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    margin-top: 1em;
+    height: 200px;
+    z-index: 1;
+}
+
+.squiggle-line {
+    height: 150%;
+}
+
+/* --------------- 4.4 Main-About-section ------------------- */
+
+.main__inner-container {
```

Comment:
Voor ieder stuk van de opdracht komt het een beetje terug op wat ik eerst vertelde. Een reusable outer en inner container die echt los staan van de andere elementen maar wel zorgen voor de structuur van de pagina scheelt al een hoop. Hier kan je dan door op bouwen, nu valt er best wel wat herhaling in je code!